Tests/Marketplace/marketplace_services.py

```diff
@@ -3007,7 +3026,212 @@ def copy_author_image(self, production_bucket, build_bucket, images_data, storag
+
+        Args:
+            production_bucket (google.cloud.storage.bucket.Bucket): The production bucket
+            build_bucket (google.cloud.storage.bucket.Bucket): The build bucket
+            images_data (dict): The images data structure from Prepare Content step
+            storage_base_path (str): The target destination of the upload in the target bucket.
+            build_bucket_base_path (str): The path of the build bucket in gcp.
+        Returns:
+            bool: Whether the operation succeeded.
+        """
+        task_status = True
+        err_msg = f"Failed copying {self._pack_name} pack integrations images."
+        pc_uploaded_readme_images = images_data.get(self._pack_name, {}).get(BucketUploadFlow.README_IMAGES, False)
+
+        if not pc_uploaded_readme_images:
+            logging.info(f"No added/modified readme images were detected in {self._pack_name} pack.")
+            return task_status
+
+        for readme_image_name in pc_uploaded_readme_images:
+            logging.info(f'copying image {readme_image_name}')
+            build_bucket_readme_image_path = os.path.join(build_bucket_base_path, self._pack_name,
+                                                          BucketUploadFlow.README_IMAGES, readme_image_name)
+            build_bucket_image_blob = build_bucket.blob(build_bucket_readme_image_path)
+
+            if not build_bucket_image_blob.exists():
+                logging.error(f"Found changed/added readme image in pack {self._pack_name} in content repo but "
+                              f"{build_bucket_image_blob} does not exist in build bucket")
+                task_status = False
+            else:
+                logging.info(f"Copying {self._pack_name} pack readme {readme_image_name} image")
+                try:
+                    copied_blob = build_bucket.copy_blob(
+                        blob=build_bucket_image_blob, destination_bucket=production_bucket,
+                        new_name=os.path.join(storage_base_path, self._pack_name, BucketUploadFlow.README_IMAGES,
+                                              readme_image_name)
+                    )
+                    if not copied_blob.exists():
+                        logging.error(
+                            f"Copy {self._pack_name} integration readme image: {build_bucket_image_blob.name} "
+                            f"blob to {copied_blob.name} blob failed.")
+                        task_status = False
+
+                except Exception as e:
+                    logging.exception(f"{err_msg}. Additional Info: {str(e)}")
+                    return False
+
+        if not task_status:
+            logging.error(err_msg)
+        else:
+            logging.success(f"Copied readme images for {self._pack_name} pack.")
+
+        return task_status
+
+    def upload_readme_images(self, storage_bucket, storage_base_path, diff_files_list=None, detect_changes=False,
+                             marketplace=''):
```

Comment:
Let's give `xsoar` as a default value for marketplace.

Reply:
Done